pkg/controller/defaults/placements.go

```diff
@@ -228,7 +228,7 @@ var (
 	}
 )
 
-func getTopologySpreadConstraintsSpec(selectorValue string) []corev1.TopologySpreadConstraint {
+func getTopologySpreadConstraintsSpec(selectorValue []string) []corev1.TopologySpreadConstraint {
```

Comment:
Rather than taking an explicit `[]string`, can you take `string...`?

Reply:
+1

Reply:
thanks, done